xds/bootstrap/bootstrap.go

```diff
@@ -0,0 +1,60 @@
+/*
+ *
+ * Copyright 2022 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Package bootstrap provides the functionality to register possible options
+// for aspects of the xDS client through the bootstrap file.
```

Comment:
Please add an experimental warning for the whole package, e.g.:

https://github.com/grpc/grpc-go/blob/980790869b005639bbf0e742f8c3008b235a1e6e/authz/rbac_translator.go#L19-L22

Reply:
Done.